plan.md

```diff
@@ -1,9 +1,282 @@
-# Plan [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/phase-two/issues)
+# The Plan [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/phase-two/issues)
 
 
 ## Why?
 
 ### “_If you fail to plan, you are planning to fail!_” ~ <small>Benjamin Franklin</small>
 
-> This plan follows _logically_ on from the **Goal**,
-so if you have not read it please do that _first_!
+
+## What?
+
+If you've been around here for a while, you may have noticed that dwyl does many things:
+we build open source [apps](https://github.com/dwyl/time), websites and [modules](https://github.com/dwyl/hapi-auth-jwt2),
+we build them for ourselves and for clients, we write tutorials, promote open source,
+provide technology training, get involved in accessibility and women in tech initiatives.
+
+It didn't start out like this.
+
+We began with [a product MVP](https://github.com/dwyl/time),
+then in order to support [Founders & Coders](foundersandcoders.com) and ensure the lights
+could be kept on at this organisation that dwyl's co-founders were heavily involved
+in building, we [started to take on client projects](https://github.com/dwyl/ateam-proposal)(again)
+whilst building our learning community by sharing the knowledge as we went.
+
+This plan lays out **how dwyl aims to achieve its goals**, _clarifying_ the many hats
+dwyl wears and how they _work together_.
+
+## Contents Guide
+For the time being, we are putting everything into this one .md file, to be split
+out once reviewed from its rough draft form. We'll need a way to navigate!
++ [Context](#context)
++ [Branches](#branches)
+  + [Product](#product)
+  + [Services](#services)
+  + [Training](#training)
+  + [Co-working](#co-working)
+
+
+## Context
+
+![tree-branches-image](http://www.m-a.org.uk/features/big/branches_main.jpg)
+
+2017 is the year of **formalising the various types of work we do into their
+own, neat groupings of activities**, the various _branches_ of dwyl.
+
+> Each **branch** will have it's own mission, detailed goals and timelines,
+feeding directly into dwyl's overall goal.
+
+dwylers will work within one branch for at least 50% of their time, but _will
+contribute to more than one_ depending on personal preferences and dwyl's needs.
+For example, a developer working mostly on product could spend some time doing QA
+on a _services_ project and also film a introductory video tutorial for a new technology they've learned.
+
+## Branches
+All branches **must contribute to our fixed overheads** and/or contribute to our
+credibility as experts in a given field.
+
+Each branch will have **its own set of goals**, derived from [dwyl's guiding principles](github.com/dwyl/phase-two#guiding-principles)
```

Comment:
I'm in _favour_ of letting someone `else` "_fix_" this "_broken link_" because it shows that they are actually paying attention and clicking through... 😉